lib/SILGen/ManagedValue.cpp

```diff
@@ -37,6 +37,24 @@ ManagedValue ManagedValue::copy(SILGenFunction &SGF, SILLocation loc) const {
   return SGF.emitManagedRValueWithCleanup(buf, lowering);
 }
 
+/// Emit an unmanaged copy of this value
+/// NOTE: Callers of this API should manage the cleanup of this value!
```

Comment:
This is a local implementation comment. I would use 2 slashes. 3 slashes is for public doxygen APIs on declarations.